### PR TITLE
GR-1428 update on target graphs

### PR DESCRIPTION
**Pull request by mengjenn**
Jira ticket or GitHub issue: [GR-1428 Update Dashi On Target graphs to use PCT_SELECTED_BASES from the HsMetrics workflow](https://jira.oicr.on.ca/browse/GR-1428)

- [x] Updates CHANGELOG.md
- [ ] Updates dev docs if applicable

### CODE REVIEW

File: application/dash_application/views/call_ready_tar.py [hunk 1]
```diff
@@ -244,6 +246,8 @@ def get_merged_ts_data():
      "value": special_cols["On Bait Percentage"]},
     {"label": "Near Bait",
      "value": special_cols["Near Bait Percentage"]},
+    {"label": "On Target", # Create scatter plot for on target reads (%)
```
Comment (avarsava) [inline application/dash_application/views/call_ready_tar.py]: leftover todo?

File: application/dash_application/views/call_ready_tar.py [hunk 2]
```diff
@@ -380,6 +384,18 @@ def generate_bait(df):
         },
     )
 
+def generate_on_target_reads_scatter_TEST_DO_NOT_USE(df, graph_params):
```
Comment (avarsava) [inline application/dash_application/views/call_ready_tar.py]: test code?

File: application/dash_application/views/single_lane_tar.py [hunk 3]
```diff
@@ -185,6 +191,8 @@ def get_bamqc_data():
      "value": special_cols["On Target Reads (%)"]},
     {"label": "Mean Insert Size",
      "value": BAMQC_COL.InsertMean},
+    {"label": "On-target Reads",
```
Comment (avarsava) [inline application/dash_application/views/single_lane_tar.py]: I think our convention is 'On Target Reads'

**Review by avarsava — COMMENTED** [review]
Whatever auto-formatted the code made the diff a bit hard to read

File: application/dash_application/views/call_ready_tar.py [hunk 4]
```diff
@@ -491,53 +508,53 @@ def layout(query_string):
...
-                    ]) # End Tabs
-                ]) # End Div
+                html.Div(className="seven columns",
+                         children=[
+                             core.Tabs([
+                                 # Graphs tab
+                                 core.Tab(label="Graphs",
+                                          children=[
+                                              create_graph_element_with_subplots(ids["graphs"], df, initial, GRAPHS),
+                                              core.Graph(
```
Comment (avarsava) [inline application/dash_application/views/call_ready_tar.py]: Need to also remove the old bar graph

**Comment by slazicoicr** [comment]
Hmm, this requires a little more thought than previously thought. I've asked for advice at https://oicr.slack.com/archives/C9P54MKB7/p1658504509636409 . It deals with https://github.com/oicr-gsi/dashi/pull/360/files#diff-3b363597b3a9835a971ddb0e80a1c15bb6be4cb2b312bf056dbe95032387caabR100 , where you try to smush merged (correctly called call ready data) into single late data. Those two data sources have different keys, so you can't merge them. I'll wait for a response from Slack to see how to proceed.

**Comment by slazicoicr** [comment]
@mengjenn The way forward is to rename the single-lane to `Estimated On Target Reads (%)` and keep the data source and calculations the same.

**Review by avarsava — APPROVED** [review]

**Review by slazicoicr — APPROVED** [review]
Update https://wiki.oicr.on.ca/pages/viewpage.action?pageId=137535526 with the changes.

**Pull request merged**